Documentation/ColumnsConfig/CommonProperties/FieldInformation/Index.rst

```diff
@@ -5,27 +5,24 @@
 fieldInformation
 ================
 
+The `fieldInformation` is a reserved area within a single form element between
+the label and the form element itself.
+
+..  include:: /Images/Rst/FieldInformationTcaDescription.rst.txt
+
 ..  confval:: fieldInformation
     :Path: $GLOBALS['TCA'][$table]['columns'][$field]['config']
     :type: array
     :Scope: Display
-    :Types: :ref:`check <columns-check>`, :ref:`flex <columns-flex>`,
-      :ref:`group <columns-group>`,
-      :ref:`imageManipulation <columns-imageManipulation>`,
-      :ref:`input <columns-input>`, :ref:`none <columns-none>`,
-      :ref:`radio <columns-radio>`
-
-    Show information between an element label and the main element input area. Configuration
-    works identical to the "fieldWizard" property, no default configuration in the core exists (yet).
-    In contrast to "fieldWizard", HTML returned by fieldInformation is limited, see
-    :ref:`FormEngine docs <t3coreapi:FormEngine-Rendering-NodeExpansion>` for more details.
-
-..  hint::
-
-    Currently, :php:`fieldInformation` is implement for all form elements,
-    except in cases where it does not make sense. You can use
-    :ref:`description <columns-properties-description>` to display general
-    information between a fields title and the form element itself.
+
+Currently, TYPO3 comes with following implemented `fieldInformation` nodes:
+
+..  toctree::
+    AdminIsSystemMaintainer
+    BackendLayoutFromParentPage
+    TcaDescription
```

Comment:
```suggestion

..  toctree::
    :titlesonly:

    AdminIsSystemMaintainer
    BackendLayoutFromParentPage
    TcaDescription
```
I would use titlesonly so that subheaders are hidden here as we only want to list the config options

Reply:
Done